src/Foundatio.Mediator/Foundatio.Mediator.props

```diff
@@ -2,5 +2,6 @@
   <ItemGroup>
     <CompilerVisibleProperty Include="DisableMediatorInterceptors" />
     <CompilerVisibleProperty Include="MediatorHandlerLifetime" />
+    <CompilerVisibleProperty Include="EnableMediatorOpenTelemetry" />
```

Comment:
Let's change this to `MediatorDisableOpenTelemetry` and make it enabled by default.